spec/CodeChain-Address.md

```diff
@@ -1,4 +1,4 @@
-When transferring CCC or assets, the sender must know the recipient's lock script hash and parameters. An address is a converted form of the lock script hash and parameters, and it has some benefits.
+When transferring CCC or addresses, the sender must know the recipient's lock script hash and parameters. An address is a converted form of the lock script hash and parameters, and it has some benefits.
```

Comment:
This part will be removed entirely when account becomes independent from the host. BTW, replacing `assets` with `addresses` is inappropriate. Let's just remove the `or assets` here for a quick fix.

Reply:
I modified it. : )